files/galaxy/config/datatypes_conf.xml

```diff
@@ -29,8 +29,8 @@
     <datatype extension="arb" auto_compressed_types="gz" type="galaxy.datatypes.binary:Binary" subclass="true" display_in_upload="true"/>
     <datatype extension="arff" type="galaxy.datatypes.text:Arff" mimetype="text/plain" display_in_upload="true"/>
     <datatype extension="paf" auto_compressed_types="gz" type="galaxy.datatypes.text:Paf" mimetype="text/plain" display_in_upload="true"/>
-    <datatype extension="gfa1" auto_compressed_types="gz" type="galaxy.datatypes.text:Gfa1" mimetype="text/plain" display_in_upload="true"/>
```

Comment:
@gallardoalba this was needed for VGP, wasn't it? Do we have upstream it?

Reply:
Yes, it is also upstream.

Reply:
but for the 21.01 release we will need it here, I guess?

Reply:
I merged it from upstream
https://github.com/galaxyproject/galaxy/blob/603122bf2a3c8308dfc1f3d2686ba9f2b2841a68/lib/galaxy/config/sample/datatypes_conf.xml.sample#L31

Reply:
@gallardoalba can you please check where this was lost? Or maybe never upstreamed.

Reply:
It is in the dev branch of the Galaxy repo https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/sample/datatypes_conf.xml.sample#L31; I thought that it means to be upstream; I guess that this is not what you were referring?

Reply:
Ah, it doesn't include the compression.

Reply:
Yes, you are right, it wasn't merged finally https://github.com/galaxyproject/galaxy/pull/13501. I cannot understand why it was closed.

Reply:
That was a mistake I think. Please reopen that PR.

Reply:
I have not closed this. In fact it doesn't say who or why the PR was closed, quite mysterious.

Reply:
> I have not closed this. In fact it doesn't say who or why the PR was closed, quite mysterious.

Thanks Marius! I'll open it again.

Reply:
Actually I can tell you why, the commit landed in dev via another PR, so there was nothing to merge as the PR was still targeting dev.